### PR TITLE
refactor: use second person for identity building

### PR DESCRIPTION
**Pull request by maciejmajek**
## Purpose

A few months ago, I have been testing different styles of system prompts. The tests included first (I am a...), second (you are ...) and third person view (This is a conversation between an expert embodied AI and user).
Empirical tests showed, that the second person view worked a little bit better than the first person and much better than the third.
Due to amount of changes in the other parts of the repository, change to second person was postponed. Until today.

## Proposed Changes

Changed the prompt of identity builder to start the robot_identity.txt with You are instead of I am.

## Issues

- Links to relevant issues

## Testing

- How was it tested, what were the results?

### CODE REVIEW

File: src/rai/rai/cli/rai_cli.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def build_robot_identity():
             "as well as the robot's capabilities and limitations including sensor and actuator information. "
             "If there are any images provided, make sure to take them into account by thoroughly analyzing them. "
             "Your description should be thorough and detailed."
-            "Your reply should start with I am a ..."
+            "Your reply should start with You are ..."
```
Comment (boczekbartek) [inline src/rai/rai/cli/rai_cli.py:99]: Just to double check, it's for identity building, but not for the identity itself, right? So the robot will still respond in the first person, like "I am a rosbot XL ..." to  "who are you question?".

(We really have to add tests to check that!)

Comment (maciejmajek) [reply src/rai/rai/cli/rai_cli.py:99]: Yes, this is only for the system prompt

Comment (boczekbartek) [reply src/rai/rai/cli/rai_cli.py:99]: Thanks! I'll test it tomorrow

**Review by boczekbartek — APPROVED** [review]
@maciejmajek 
Apologies for my late reply. 
I tested it on turtlebot and identity was adjusted as expected.
See image (left - old, right - new). 
image

I also briefly tested it with [turtlebot_demo](https://github.com/RobotecAI/rai/blob/development/src/examples/turtlebot4/README.md) and embodiment works.

**Comment by maciejmajek** [comment]
@boczekbartek actually that diff is kind of strange. I've tested it and the output always started with "you are a" which is vastly different from "you are looking at". I will further adjust it as at the moment this may not propagate embodiment well

**Comment by boczekbartek** [comment]
I see. However, when I tested it the answers in streamlit were in the first person. 

>  I will further adjust it as at the moment this may not propagate embodiment well

Sure!

**Pull request merged**